src/library/header/LibraryHeader.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Route, Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { Dropdown, Icon } from '@edx/paragon';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { AppContext } from '@edx/frontend-platform/react';
+import { ensureConfig } from '@edx/frontend-platform/config';
+
+import { libraryShape } from '../common';
+import { selectLibraryDetail } from '../detail';
+import StudioLogo from './assets/studio-logo.png';
+import messages from './messages';
+
+ensureConfig([
+  'CMS_BASE_URL',
+  'LOGOUT_URL',
+], 'Library header');
+
+class LibraryHeader extends React.Component {
```

Comment:
How about `LibraryAuthoringHeader`? Someone unfamiliar might guess that `LibraryHeader` refers to the header of an individual library.

Reply:
I called it StudioHeader, since that's what it is, or at least what it's intended to be. :shrug: